src/lib/components/FiltersSection/index.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview handles the events for a modal containing filters
+ */
+import {BaseElement} from '../BaseElement';
+
+import {clearFilters} from '../../actions';
+
+export class FiltersSection extends BaseElement {
```

Comment:
Noticed that the filters layout is a bit different from DCC, so I transformed filters-modal into a more generic filters-section